README.md

```diff
@@ -119,42 +119,52 @@ All designs are in [Figma](https://www.figma.com/files/team/834568130405102661/H
 
 ### File and directory structure conventions
 
-<details><summary>For React</summary>
+<details><summary>:atom_symbol: React</summary>
+
+> **Note**  
+> **Any mentions of `.js`/`.jsx` in the following guidelines are interchangeable with `.ts`/`.tsx`. New features should ideally be developed in TypeScript.**
 
 - Components with tests or styles should be placed in to their own directory.
-- Styles should follow the `style.*` pattern where the extension is js for styled components or css for stylesheets.
-- Tests should follow the `*.spec.js` pattern...
-- and stories should follow the `*.stories.js` pattern. For both, the prefix is the name of the component.
+- Styles should follow the `style.*` pattern where the extension is `js` for styled components or `css` for stylesheets.
+  - New styled components should use `styled` from `@mui/styles`.
+- Supporting test files have specific naming conventions:
+  - Jest Tests should follow the `*.spec.js` pattern.
+  - Stories should follow the `*.stories.js` pattern.
+  - Cypress tests should follow the `*.cy.js` pattern.
+  - For all test files, the prefix is the name of the component.
 - Each component directory should have an `index.js` which exports the component as default.
 - Components which share a common domain can be placed in a directory within components named after the domain.
 
 </details>
 
-<details><summary>For images</summary>
+<details><summary>:framed_picture: Images</summary>
 
-Images should displayed using the `source srcset` attribute. You should prepare four versions of the image starting at its original size and at 75%, 50% and 25% the original image's size preserving its aspect ratio. If available, you should also provide a 2x resolution for higher density screens. For example, to resize images using Mac's Preview you can visit the 'Tools' menu and select 'Adjust Size', from there you can change the image's width while making sure 'Scale Proportionally' and 'Resample Image' are checked. Once ready, each version of the image should be processed with an image optimizer such as ImgOptim.
+Images should displayed using the `source srcset` attribute. You should prepare four versions of the image starting at its original size and at 75%, 50% and 25% the original image's size preserving its aspect ratio. If available, you should also provide a 2x resolution for higher density screens. 
+- For example, to resize images using Mac's Preview you can visit the 'Tools' menu and select 'Adjust Size', from there you can change the image's width while making sure 'Scale Proportionally' and 'Resample Image' are checked. Once ready, each version of the image should be processed with an image optimizer such as [ImgOptim](https://imageoptim.com/mac) or [Online Image Compressor](https://imagecompressor.com/).
 
 Finally after processing, the images should be added to the S3 bucket, `portal-ui-images-s3-origin`,
```

Comment:
Should we add something here about invalidating the CloudFront cache if updating an image? The default max age is 180 days.

Reply:
I'll add this section, I think that'd be really useful!

Reply:
I added the relevant info right after the section that mentions the `cache-control` header a bit further down 👍🏻